services/121-service/src/user/user.service.spec.ts

```diff
@@ -0,0 +1,439 @@
+import { HttpStatus } from '@nestjs/common';
+import { HttpException } from '@nestjs/common/exceptions/http.exception';
+import { REQUEST } from '@nestjs/core';
+import { Test, TestingModule } from '@nestjs/testing';
+import { getRepositoryToken } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+
+import { ProgramEntity } from '@121-service/src/programs/entities/program.entity';
+import { ProgramAidworkerAssignmentEntity } from '@121-service/src/programs/entities/program-aidworker.entity';
+import { PermissionEntity } from '@121-service/src/user/entities/permissions.entity';
+import { UserEntity } from '@121-service/src/user/entities/user.entity';
+import { UserRoleEntity } from '@121-service/src/user/entities/user-role.entity';
+import { UserService } from '@121-service/src/user/user.service';
+import { UserEmailsService } from '@121-service/src/user/user-emails/user-emails.service';
+
+describe('UserService', () => {
+  let service: UserService;
+  let userRepository: Repository<UserEntity>;
+
+  let permissionRepository: Repository<PermissionEntity>;
+
+  let userRoleRepository: Repository<UserRoleEntity>;
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars -- used in tests
+  let programRepository: Repository<ProgramEntity>;
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars -- used in tests
+  let assignmentRepository: Repository<ProgramAidworkerAssignmentEntity>;
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars -- used in tests
+  let userEmailsService: UserEmailsService;
```

Comment:
These variables (`programRepository`, `assignmentRepository`, `userEmailsService`) are declared but never actually used in any of the test methods, despite the eslint-disable comments claiming they're "used in tests". We can probably clean up the code by removing these unused variables